src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -16,17 +19,17 @@ public static void main(String[] args) {
         SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
 
         Country usa = new Country("USA");
-        CountryService countryService = null; // TODO: initialize this instance
+        CountryService countryService = new CountryServiceImpl();
         countryService.add(usa);
 
         Actor vinDiesel = new Actor("Vin Diesel");
         vinDiesel.setCountry(usa);
-        ActorService actorService = null; // TODO: initialize this instance
+        ActorService actorService = new ActorServiceImpl();
         actorService.add(vinDiesel);
 
         Movie fastAndFurious = new Movie("Fast and Furious");
         fastAndFurious.setActors(List.of(vinDiesel));
-        MovieService movieService = null; // TODO: initialize this instance
+        MovieService movieService = new MovieServiceImpl();
```

Comment:
The `MovieServiceImpl` should be initialized with the `sessionFactory` to ensure it can manage Hibernate sessions correctly. Modify the constructor to accept a `SessionFactory` parameter.